syft/generic/frameworks/hook/hook.py

```diff
@@ -16,7 +16,6 @@
 from syft.generic.frameworks.hook.pointers import PointerHook
 from syft.generic.frameworks.hook.string import StringHook
 from syft.generic.frameworks.hook.tensors import TensorHook
-from syft.generic.pointers.pointer_tensor import PointerTensor
```

Comment:
Great finding! While we are here may I ask yo to put together (line by line) all the syft import and to sort them alphabetically? We've tried to do this in most of our files but apparently here it wasn't done^^

Reply:
complete!